sdk/Lusid.Sdk.Tests/tutorials/Instruments/SimpleInstrumentExample.cs

```diff
@@ -4,21 +4,23 @@
 using Lusid.Sdk.Api;
 using Lusid.Sdk.Model;
 using Lusid.Sdk.Tests.Utilities;
+using LusidFeatures;
 using NUnit.Framework;
 
 namespace Lusid.Sdk.Tests.Tutorials.Instruments
 {
     [TestFixture]
-    public class EquityExample: DemoInstrumentBase
+    public class SimpleInstrumentExample: DemoInstrumentBase
     {
+        [LusidFeature("F5-21")]
         [Test]
-        public void EquityCreationAndUpsertionExample()
+        public void SimpleInstrumentCreationAndUpsertionExample()
         {
             // DEFINE scope
             string scope = "ibor";
             
             // CREATE a Simple Instrument Equity (that can then be upserted into LUSID)
-            var equity = (SimpleInstrument) InstrumentExamples.CreateExampleEquity();
+            var equity = (SimpleInstrument) InstrumentExamples.CreateExampleSimpleInstrument();
```

Comment:
perhaps cast with `as` so it doesn't throw given you are asserting the line below